From f4795bf64dbf8778fc765b7dad1b1bfb365b48d7 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Wed, 8 May 2024 19:54:16 +0200
Subject: configure.ac: fix sixaxis build without tools

Fix the following build failure with --enable-sixaxis --enable-udev
--disable-tools raised since version 5.73 and
9f71892b63f6b65dab874a633d88ae2dcad93a16:

sixaxis.c:(.text.sixaxis_init+0x23): undefined reference to `udev_new'

Fixes: http://autobuild.buildroot.org/results/c337d0f473c5fbb3e6cda1317ba06f5e2b16a43e

Upstream: https://git.kernel.org/pub/scm/bluetooth/bluez.git/commit/?id=f4795bf64dbf8778fc765b7dad1b1bfb365b48d7
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 9dea70ddc1..8b3cdb4e8f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -226,7 +226,7 @@ AM_CONDITIONAL(MONITOR, test "${enable_monitor}" != "no")
 
 AC_ARG_ENABLE(udev, AS_HELP_STRING([--disable-udev],
 		[disable udev device support]), [enable_udev=${enableval}])
-if (test "${enable_tools}" != "no" && test "${enable_udev}" != "no"); then
+if (test "${enable_udev}" != "no"); then
 	PKG_CHECK_MODULES(UDEV, libudev >= 196)
 	AC_DEFINE(HAVE_UDEV, 1, [Define to 1 if udev is required])
 fi
-- 
cgit 1.2.3-korg

